lib/datadog/tracing/contrib/active_support/cache/instrumentation.rb

```diff
@@ -58,7 +58,8 @@ def trace(action, store, key: nil, multi_key: nil)
                 end
 
                 span.set_tag(Ext::TAG_CACHE_BACKEND, store) if store
-                set_cache_key(span, key, multi_key)
+
+                set_cache_key(span, key, multi_key) if Datadog.configuration.tracing[:active_support][:cache_key_enabled]
```

Comment:
```suggestion
                set_cache_key(span, key, multi_key) if configuration[:cache_key_enabled]
```

Reply:
@marcotc this seems to be not work (I'm likely doing something wrong)

```
      Failure/Error: set_cache_key(span, key, multi_key) if configuration[:cache_key_enabled]
      
      NameError:
        undefined local variable or method `configuration' for Datadog::Tracing::Contrib::ActiveSupport::Cache::Instrumentation:Module
```